packages/components/src/Tooltip/Tooltip.mdx

```diff
@@ -32,6 +32,19 @@ import { Tooltip } from "@jobber/components/Tooltip";
   </Tooltip>
 </Playground>
 
+## Design & Usage Guidelines
+
+When contributing to, or consuming the Tooltip component, consider the following:
+
+### Usage
+- A tooltip should be used to introduce a novel UI element or highlight a change to an existing interface.
+- A tooltip should contain information that is helpful, but not necessary for the user to successfully interact with the page.
+- A tooltip should not be a permanent fixture in the interface - if you need a message that remains in the view, consider putting that content inline. 
+
+### Content
+- Tooltip content should consist of only text and an optional "dismiss" CTA
+- Tooltip text content should not exceed 3 lines
```

Comment:
ty for specifying length, we've got some long bois out there 🙏

Reply:
l o o o n g b o i s